test/unit/errors/error-collector.test.js

```diff
@@ -1675,6 +1676,30 @@ test('Errors', async (t) => {
     })
   })
 
+  await t.test('error events', async (t) => {
+    t.beforeEach(beforeEach)
+    t.afterEach(afterEach)
+
+    await t.test('should use passed in segment', (t) => {
```

Comment:
this is fine because it's asserting the segment getting passed in is being used. it looks like there's a test in `error_events` that asserts this on the active segment.